src/app/virtrolio-cover/virtrolio-cover.component.css

```
@@ -0,0 +1,43 @@
+#virtrolio-outer-div {
+    /* Div enclosing virtrolio (outside <a> tag) */
+    height: 35em;
+    padding: 0 4.5%;
+}
+
+#virtrolio-outer {
+    /* Div enclosing virtrolio (inside <a> tag) */
+    height: 100%;
+}
+
+#virtrolio-title {
+    /* Title of the virtrolio (currently 2019-2020) */
+    height: 10em;
+}
+
+#usersVirtrolio {
+    /* "User's virtrolio" text on virtrolio cover */
+    text-align: center;
+    color: white;
+}
+
+#friend-link-body {
+    /* Contents of friend-link modal body */
+    text-align: center;
+}
+
+#friend-link-body > * {
+    /* Each item inside the friend-link modal body */
+    margin: 1em;
+}
+
+#friend-link-field {
+    /* Friend link text field (where the link is) */
+    width: 15em;
+}
+
+#buttons > *.row > * {
+    /* Each individual button will have these properties: */
+    height: 4em;
+    width: 100%;
+    margin: 1.2em;
+}
```

No newline at end of file
```suggestion
}

```